### ISSUE TITLE
Just a humble request to do an AMA

### ISSUE THREAD

**Issue by Thewisem** [KaustubhPatange/Gear-VPN#4]
You could do a reddit AMA on privaytools.io or any vpn subreddit. It helps the community know who is running this and also gives your vpn some exposure.

**Comment by Thewisem** [comment]
AMA is Ask ms anything

**Comment by KaustubhPatange** [comment]
I did an AMA on r/androidapps a while ago. I know about r/VPN but I don't think they would allow me to post about this app on their subreddit. Do you know any other subreddits I can make a post about this app?

**Comment by Thewisem** [comment]
Privacty tools.io Subreddit. Its your best bet at reaching target audience and people who actually care about privacy

**Comment by KaustubhPatange** [comment]
There are currently some issues with the app which are causing some weird crashes on different devices. Once the app becomes stable (which shouldn't take a long time) I'll start promoting it. Thanks for the info!

**Comment by yesNOmaybe69420pogchamp69** [comment]
Do an ama on Lemmy instead of reddit

**Comment by Thewisem** [comment]
Trust me when I say more people will see on reddit than Lemmy and more people would ask. But you could do it on Lemmy. Don't stick to a single platform

**Comment by KaustubhPatange** [comment]
Yes, will do an AMA soon (maybe in the next week) currently there are some feature requests and a lot of bugs that are needed to be fixed so that the app becomes more stable and I don't have to worry about constant app crash issues popping on my dashboard.

**Comment by gdnzz** [comment]
Appearance on privacytools.io would be best exposure for your app.
